src/frontend/web_application/src/components/NewDraftForm/style.scss

```diff
@@ -24,29 +24,24 @@
     border-left: 1px solid $co-color__bg__back;
   }
 
-  &__type {
-    @include flex-grid-size(shrink);
-  }
-
+  &__type { @include flex-grid-size(shrink); }
   &__type-icon { color: $co-color__fg__text; }
-
-  &__author {
-    @include flex-grid-column(shrink);
-    color: $co-color__fg__text--high;
-    font-weight: 600;
-  }
+  &__type-label { display: none; }
```

Comment:
it sounds `__type-label` is not overridden by a breakpoint.
So for what is it useful for ?

Reply:
it is overridden by breakpoint(medium): `&__type-label { display: inline; }` :)